Tests/Expected/Storyboards-iOS/default-context-all-customname.swift

```diff
@@ -1,5 +1,6 @@
 // Generated using SwiftGen, by O.Halligon — https://github.com/SwiftGen/SwiftGen
 
+// swiftlint:disable sorted_imports
```

Comment:
Oups sorry my bad, I was deep in the review and did not see it was a 'expected' file 😅